dfinity/nns-dapp
# Feat: Improve DisburseMaturityModal information

# Motivation

Improve the information we show to the user when disbursing maturity.

# Changes

* Improve message in the first step of DisburseMaturityModal.
* Add data in the second step of DisburseMaturityModal.

# Tests

* Add tests that the new information is present as expected.
* Add methods in relevant POs.

# Todos

Not necessary yet, it's covered by the disburse maturity changelog entry.
